content/chainguard/libraries/python/overview.md

```diff
@@ -77,6 +77,11 @@ Alternatively, you can use the token for direct access to the Chainguard
 Libraries for Python index as discussed in [Build
 configuration](/chainguard/libraries/python/build-configuration/).
 
+Follow the steps detailed in [Manual Access](#manual) to browse the
+simple Python index and find available packages, package versions, source
```

Comment:
What is the meaning of "simple" here?

Reply:
If it's just a qualitative thing, I would remove it

Reply:
Its the technical term of the Python index that is browsable .. I can remove here though